internal/controllers/cryostat_controller.go

```diff
@@ -316,6 +316,12 @@ func (r *CryostatReconciler) Reconcile(ctx context.Context, request ctrl.Request
 		}
 	}
 
+	instance.Status.GrafanaSecret = instance.Name + "-grafana-basic"
```

Comment:
Perhaps for this, we could use the value of `grafanaSecret.Name`? It'll be less error-prone if we decide to change the secret's name in the future.